pyfar/constants/air_attenuation.py

```diff
@@ -0,0 +1,243 @@
+    if np.any(np.array(atmospheric_pressure) > 200000):
+        raise ValueError("Atmospheric pressure must less than 200 kPa.")
+
+    # convert arrays
+    all_are_singles = isinstance(temperature, (float, int))
+    all_are_singles = all_are_singles and isinstance(frequency, (float, int))
+    all_are_singles = all_are_singles and isinstance(
+        relative_humidity, (float, int))
+    if not all_are_singles:
+        shape = np.broadcast_shapes(
+            np.array(temperature).shape,
+            np.array(frequency).shape,
+            np.array(relative_humidity).shape,
+            np.array(atmospheric_pressure).shape)
+        temperature = np.broadcast_to(
+            np.array(temperature, dtype=float), shape)
+        frequency = np.broadcast_to(
+            np.array(frequency, dtype=float), shape)
+        relative_humidity = np.broadcast_to(
+            np.array(relative_humidity, dtype=float), shape)
+        atmospheric_pressure = np.broadcast_to(
+            np.array(atmospheric_pressure, dtype=float), shape)
+
+    # calculate air attenuation
+    p_atmospheric_ref = 101325
+    t_degree_ref = 20
+
+    p_a = atmospheric_pressure
+    p_r = p_atmospheric_ref
+    f = frequency
+    T = temperature + 273.15
+    T_0 = t_degree_ref + 273.15
+
+    p_sat_water = saturation_vapor_pressure(temperature)
+    p_vapor = relative_humidity*p_sat_water
+
+    # molar concentration of water vapor as a percentage
+    h = p_vapor/p_a*10000
+
+    # Oxygen relaxation frequency (Eq. 3)
+    f_rO = (p_a/p_r)*(24+4.04e4*h*(0.02+h)/(0.391+h))
+
+    # Nitrogen relaxation frequency (Eq. 4)
+    f_rN = (p_a/p_r)*(T/T_0)**(-1/2)*(9+280*h*np.exp(
+        -4.17*((T/T_0)**(-1/3)-1)))
+
+    # air attenuation (Eq. 5)
+    air_attenuation = 8.686*f**2*((1.84e-11*p_r/p_a*(T/T_0)**(1/2)) + \
+        (T/T_0)**(-5/2)*(0.01275*np.exp(-2239.1/T)*(f_rO + (f**2/f_rO))**(-1)
+        +0.1068*np.exp(-3352/T) * (f_rN + (f**2/f_rN))**(-1)))
+
+    if not calculate_accuracy:
+        return air_attenuation
+
+    # calculate accuracy
+    accuracy = np.zeros_like(np.atleast_1d(air_attenuation)) - 1
+    freq2pressure = frequency/atmospheric_pressure
+    vapor_1_mask = 0.05 <= p_vapor <= 5
+    vapor_2_mask = not vapor_1_mask and 0.005 <= p_vapor
+    vapor_3_mask = 0.005 > p_vapor
+    temp_1_mask = -20 <= temperature <= 50
+    temp_2_mask = -70 <= temperature and not temp_1_mask
+    atm_mask = atmospheric_pressure <= 200000
+    freq2pressure_mask = 4e-4 <= freq2pressure <= 10
+    common_mask = atm_mask and freq2pressure_mask
+    accuracy_10 = vapor_1_mask and temp_1_mask and common_mask
+    accuracy_20 = vapor_2_mask and temp_1_mask and common_mask
+    accuracy_50 = vapor_3_mask and temp_2_mask and common_mask
+    accuracy[accuracy_10] = 10
+    accuracy[accuracy_20] = 20
+    accuracy[accuracy_50] = 50
+
+    return air_attenuation, accuracy
+
+
+def _p_sat_water(temperature):
```

Comment:
this is was replaced by saturation_vapor_pressure and is not used anymore, its just temporary here and is comping from #503